README.md

```
@@ -113,71 +164,172 @@ curl -sSL https://install.python-poetry.org | python3 -
+  }'
 ```
 
-### Web Deployment
+### Production Deployment Considerations
 
-For more than 2 agents, deploy to a web server to avoid Ngrok limitations.
+When deploying to production, always set the `BASE_URL` environment variable to ensure reliable WebSocket connections:
+
+1. Set `BASE_URL` to your server's public domain:
+
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)